src/containers/my-cooperations/cooperation-notes/CooperationNotes.styles.ts

```diff
@@ -0,0 +1,24 @@
+import palette from '~/styles/app-theme/app.pallete'
+
+export const styles = {
+  divider: {
+    mx: '16px',
+    border: `1px solid ${palette.primary[200]}`
+  },
+  notesWrapper: (isNotes: boolean) => ({
+    width: isNotes ? { md: '400px', sm: '380px' } : '0px'
```

Comment:
same here, with drawer I guess you will not need isNotes param

Reply:
> same here, with drawer I guess you will not need isNotes param

used AppDrawer instead isNotes param